docs/proposals/scheduling/multi-scheduling-group/README.md

```diff
@@ -0,0 +1,366 @@
+multiple affinity terms in `.spec.placement` of `PropagationPolicy`.
+```yaml
+// Placement represents the rule for select clusters.
+type Placement struct {
+	// ClusterAffinity represents scheduling restrictions to a certain set of clusters.
+	// If not set, any cluster can be scheduling candidate.
+	// +optional
+	ClusterAffinity *ClusterAffinity `json:"clusterAffinity,omitempty"`
+
+	// OrderedClusterAffinities represents the scheduling order among multiple
+	// set of clusters that indicated by OrderedClusterAffinityTerm.
+	// Each term should have a unique order among all terms.
+	//
+	// The scheduler will evaluate these groups one by one in strict order, the
+	// group that does not satisfy scheduling restrictions will be ignored that
+	// means all clusters in this group will not be selected unless it also
+	// belongs to the next group(a cluster could belong to multiple groups).
+	//
+	// If none of the groups satisfy the scheduling restrictions, then scheduling
+	// fails, which means no cluster will be selected.
+	//
+	// Note: OrderedClusterAffinities can not co-exist with ClusterAffinity.
+	//
+	// Potential use case 1:
+	// The private clusters in the local data center could be the main group, and
+	// the managed clusters provided by cluster providers could be the secondary
+	// group. So that the Karmada scheduler would prefer to schedule workloads
+	// to the main group and the second group will only be considered in case of
+	// the main group does not satisfy restrictions(like, lack of resources).
+	//
+	// Potential use case 2:
+	// For the disaster recovery scenario, the clusters could be organized to
+	// primary and backup groups, the workloads would be scheduled to primary
+	// clusters firstly, and when primary cluster fails(like data center power off),
+	// Karmada scheduler could migrate workloads to the backup clusters.
+	//
+	// +optional
+	OrderedClusterAffinities []OrderedClusterAffinityTerm `json:"orderedClusterAffinities,omitempty"`
+
+	// ClusterTolerations represents the tolerations.
+	// +optional
+	ClusterTolerations []corev1.Toleration `json:"clusterTolerations,omitempty"`
+
+	// SpreadConstraints represents a list of the scheduling constraints.
+	// +optional
+	SpreadConstraints []SpreadConstraint `json:"spreadConstraints,omitempty"`
+
+	// ReplicaScheduling represents the scheduling policy on dealing with the number of replicas
+	// when propagating resources that have replicas in spec (e.g. deployments, statefulsets) to member clusters.
+	// +optional
+	ReplicaScheduling *ReplicaSchedulingStrategy `json:"replicaScheduling,omitempty"`
+}
+// OrderedClusterAffinityTerm selects a set of clusters and indicates the order
+// that the Karmada scheduler should inspect.
+type OrderedClusterAffinityTerm struct {
+	ClusterAffinity `json:",inline"`
+
+	// Order of the cluster group that the Karmada scheduler should inspect with, in the range 1-100.
```

Comment:
A higher value indicates a higher priority or a lower priority? Shouldn't this be explained here?

And can users set the same value for different terms?